examples/plane/limiters_advection.jl

```diff
@@ -211,49 +215,56 @@ for (k, ne) in enumerate(ne_seq)
         end
 
         # Compute hyperviscosity for the tracer equation by splitting it in two diffusion calls
+        ystar = similar(y)
         @. ystar.ρq = wdiv(grad(y.ρq / y.ρ))
-        Spaces.weighted_dss!(ystar)
+        Spaces.weighted_dss!(ystar.ρq)
         @. ystar.ρq = -D₄ * wdiv(y.ρ * grad(ystar.ρq))
 
         # Add advective flux divergence
-        @. ystar.ρ = -wdiv(y.ρ * u)         # contintuity equation
-        @. ystar.ρq += -wdiv(y.ρq * u)      # adevtion of tracers equation
-    end
+        @. dy.ρ = beta * dy.ρ - alpha * wdiv(y.ρ * u)         # contintuity equation
+        @. dy.ρq = beta * dy.ρq - alpha * wdiv(y.ρq * u) + alpha * ystar.ρq   # adevtion of tracers equation
```

Comment:
```suggestion
        @. dy.ρq = beta * dy.ρq - alpha * wdiv(y.ρq * u) + alpha * ystar.ρq   # advection of tracers equation
```

Reply:
I guess the typo re-appeared in the merge/rebase

Reply:
@simonbyrne is there anywhere that this interface with the `alpha` and `beta` is documented? I can't find it anywhere in the ClimaTimeSteppers.jl docs and I'd like to understand what we are doing here.